### PR TITLE
Add handling of missing devices to HomematicIP Cloud

### PR DESCRIPTION
**Pull request by SukramJ**
**Description:**
Add handling of missing devices to HomematicIP Cloud

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/homematicip_cloud.markdown [hunk 1]
```diff
@@ -162,6 +162,18 @@ Within this delay the device registration should be completed in the App, otherw
   * Weather Sensor – plus (*HmIP-SWO-PL*)
   * Weather Sensor – pro (*HmIP-SWO-PR*)
   
+## What to do, if a device is missing in Home-Assistant
```
Comment (frenck) [inline source/_integrations/homematicip_cloud.markdown]:
```suggestion
## What to do, if a device is missing in Home Assistant
```

File: source/_integrations/homematicip_cloud.markdown [hunk 2]
```diff
@@ -162,6 +162,18 @@ Within this delay the device registration should be completed in the App, otherw
...
+
+In order for a device to be integrated in the Home-Assistant, this must first be implemented in the upstream library. A dump of your configuration is required for this, which is then attached to a new issue in the [upstream lib's](https://github.com/coreGreenberet/homematicip-rest-api) github repository.
```
Comment (frenck) [inline source/_integrations/homematicip_cloud.markdown]:
```suggestion
In order for a device to be integrated into Home Assistant, it must first be implemented in the upstream library. A dump of your configuration is required for this, which is then attached to a new issue in the [upstream lib's](https://github.com/coreGreenberet/homematicip-rest-api) GitHub repository.
```

File: source/_integrations/homematicip_cloud.markdown [hunk 3]
```diff
@@ -162,6 +162,18 @@ Within this delay the device registration should be completed in the App, otherw
...
+
+1.	Create a dump of your access point configuration in Home-Assistant:
```
Comment (frenck) [inline source/_integrations/homematicip_cloud.markdown]:
```suggestion
1. Create a dump of your access point configuration in Home Assistant: 
```

File: source/_integrations/homematicip_cloud.markdown [hunk 4]
```diff
@@ -162,6 +162,18 @@ Within this delay the device registration should be completed in the App, otherw
   * Weather Sensor – plus (*HmIP-SWO-PL*)
   * Weather Sensor – pro (*HmIP-SWO-PR*)
   
+## What to do, if a device is missing in Home-Assistant
+
+In order for a device to be integrated in the Home-Assistant, this must first be implemented in the upstream library. A dump of your configuration is required for this, which is then attached to a new issue in the [upstream lib's](https://github.com/coreGreenberet/homematicip-rest-api) github repository.
+
+1.	Create a dump of your access point configuration in Home-Assistant: 
+Developer Tools -> Services -> Select `homematicip_cloud.dump_hap_config` -> Execute.
```
Comment (frenck) [inline source/_integrations/homematicip_cloud.markdown]:
```suggestion
  Developer Tools -> Services -> Select `homematicip_cloud.dump_hap_config` -> Execute. 
```

File: source/_integrations/homematicip_cloud.markdown [hunk 5]
```diff
@@ -162,6 +162,18 @@ Within this delay the device registration should be completed in the App, otherw
   * Weather Sensor – plus (*HmIP-SWO-PL*)
   * Weather Sensor – pro (*HmIP-SWO-PR*)
   
+## What to do, if a device is missing in Home-Assistant
+
+In order for a device to be integrated in the Home-Assistant, this must first be implemented in the upstream library. A dump of your configuration is required for this, which is then attached to a new issue in the [upstream lib's](https://github.com/coreGreenberet/homematicip-rest-api) github repository.
+
+1.	Create a dump of your access point configuration in Home-Assistant: 
+Developer Tools -> Services -> Select `homematicip_cloud.dump_hap_config` -> Execute. 
+The default dump is anonymized and is written to your config directory (`hmip_config_XXXX.json`)
```
Comment (frenck) [inline source/_integrations/homematicip_cloud.markdown]:
```suggestion
  The default dump is anonymized and is written to your configuration directory (`hmip_config_XXXX.json`).
```

File: source/_integrations/homematicip_cloud.markdown [hunk 6]
```diff
@@ -162,6 +162,18 @@ Within this delay the device registration should be completed in the App, otherw
...
+2.  Create a [new issue](https://github.com/coreGreenberet/homematicip-rest-api/issues/new) in this github repository and attach the dump file.
```
Comment (frenck) [inline source/_integrations/homematicip_cloud.markdown]:
```suggestion
2. Create a [new issue](https://github.com/coreGreenberet/homematicip-rest-api/issues/new) at this GitHub repository and attach the created dump file.
```

File: source/_integrations/homematicip_cloud.markdown [hunk 7]
```diff
@@ -162,6 +162,18 @@ Within this delay the device registration should be completed in the App, otherw
...
+
+Please be patient and wait for the implementation and a new release of the upstream library.
```
Comment (frenck) [inline source/_integrations/homematicip_cloud.markdown]:
```suggestion
Please be patient, wait for the implementation and a new release of the upstream library.
```

File: source/_integrations/homematicip_cloud.markdown [hunk 8]
```diff
@@ -162,6 +162,18 @@ Within this delay the device registration should be completed in the App, otherw
...
+Afterwards this device can be implemented in Home-Assistant.
```
Comment (frenck) [inline source/_integrations/homematicip_cloud.markdown]:
```suggestion
Afterward, this device can be implemented into Home Assistant.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @SukramJ!

I've reviewed it, and left some comments. Could you please take a look? Thanks! 👍

**Comment by SukramJ** [comment]
Thanks a lot @frenck.

**Review by frenck — APPROVED** [review]
Thanks, @SukramJ! 👍

**Pull request merged**